http/codegen/server.go

```diff
@@ -464,6 +464,8 @@ func {{ .HandlerInit }}(
 			}
 			return
 		}
+	{{- else }}
```

Comment:
This could be moved below the `{{ if .ServerStream }}` below since the other case already defines `err`:
```go
{{ if .ServerStream }}
	var err error
	var cancel context.CancelFunc
	// ...
```